### PR TITLE
Fix isPlotOwner() on modern Towny versions.

### PR DESCRIPTION
**Pull request by LlmDl**
Towny no longer throws NotRegisteredExceptions for TownyAPI.getInstance().getTownBlock(location), which requires that we check if the location is not in the wilderness.

### CODE REVIEW

File: src/main/java/com/acrobot/chestshop/towny/Permission.java [hunk 1]
```diff
@@ -22,10 +22,6 @@ public static boolean has(Player player, String node) {
         return player.hasPermission(node) || player.hasPermission(node.toLowerCase());
     }
 
-    private static boolean hasPermissionSet(Player p, String perm) {
-        return p.isPermissionSet(perm) && p.hasPermission(perm);
-    }
-
```
Comment (Phoenix616) [inline src/main/java/com/acrobot/chestshop/towny/Permission.java:28]: Why was this removed? Don't think that should be removed even if it isn't used anywhere anymore as it's not related to the issue solved by the commit at all...

Comment (LlmDl) [reply src/main/java/com/acrobot/chestshop/towny/Permission.java:28]: I should have noted, it was unused by that class. Not sure how long it has been unused for.

Comment (Phoenix616) [reply src/main/java/com/acrobot/chestshop/towny/Permission.java:28]: Hm, ok, I guess that can be removed now. Took a look if it may be a bug that it's unused but it doesn't look like it to me. Not sure what that was meant to be used for.

**Comment by Phoenix616** [comment]
Thanks!

**Comment by LlmDl** [comment]
> Thanks!

Thanks for the quick response!
